src/sentry/utils/functional.py

```diff
@@ -39,3 +41,13 @@ def apply_values(function, mapping):
             function(values),
         ),
     )
+
+
+def compact(_dict):
+    """
+    Removes keys from a dictionary that have a ``None`` value.
+
+    >>> compact({'foo': 'bar', 'baz': None})
+    {'foo': 'bar'}
+    """
+    return {k: v for k, v in six.iteritems(_dict) if v}
```

Comment:
Won't this also slice out `False`, `0` and `''`?

Reply:
Ugh, yea. I'll learn to Python someday. Thanks!